src/symmetric_crypto/key_wrapping_rfc_5649.rs

```diff
@@ -0,0 +1,498 @@
+use aead::{generic_array::GenericArray, KeyInit};
+use aes_gcm::aes::{Aes128, Aes192, Aes256};
+use chacha20::cipher::{BlockDecrypt, BlockEncrypt};
+
+use crate::CryptoCoreError;
+
+#[allow(dead_code)]
+const DEFAULT_IV: u64 = 0xA6A6_A6A6_A6A6_A6A6;
+#[allow(dead_code)]
+const DEFAULT_RFC5649_CONST: u32 = 0xA659_59A6_u32;
```

Comment:
Idem for all functions.
```suggestion
const DEFAULT_IV: u64 = 0xA6A6_A6A6_A6A6_A6A6;
const DEFAULT_RFC5649_CONST: u32 = 0xA659_59A6_u32;
```